dissectors/pro-dj-link-status.lua

```diff
@@ -121,13 +294,93 @@ function p_pdj_status.dissector (buf, pkt, root)
     subtree_flags:add(pdj_status_f.flags_onair, flags_ptr)
     subtree_flags:add(pdj_status_f.flags_bpm, flags_ptr)
 
+    local pitch1_ptr = buf(0x8c,4)
+    local pitch1 = pitch1_ptr:uint()
+    local pitch1_display = ((pitch1 - 0x100000) / 0x100000) * 100
+    subtree:add(pdj_status_f.t0a_pitch1, pitch1_ptr, pitch1, nil, "(" .. pitch1_display .. "%)")
+
+    local pitch2_ptr = buf(0x98,4)
+    local pitch2 = pitch2_ptr:uint()
+    local pitch2_display = ((pitch2 - 0x100000) / 0x100000) * 100
+    subtree:add(pdj_status_f.t0a_pitch2, pitch2_ptr, pitch2, nil, "(" .. pitch2_display .. "%)")
+
+    local pitch3_ptr = buf(0xc0,4)
+    local pitch3 = pitch3_ptr:uint()
+    local pitch3_display = ((pitch3 - 0x100000) / 0x100000) * 100
+    subtree:add(pdj_status_f.t0a_pitch3, pitch3_ptr, pitch3, nil, "(" .. pitch3_display .. "%)")
+
+    local pitch4_ptr = buf(0xc4,4)
+    local pitch4 = pitch4_ptr:uint()
+    local pitch4_display = ((pitch4 - 0x100000) / 0x100000) * 100
+    subtree:add(pdj_status_f.t0a_pitch4, pitch4_ptr, pitch4, nil, "(" .. pitch4_display .. "%)")
+
+    local bpm_ptr = buf(0x92,2)
+    local bpm = bpm_ptr:uint()
+    local bpm_display = bpm / 100
+    subtree:add(pdj_status_f.t0a_bpm, bpm_ptr, bpm, nil, "(" .. bpm_display .. " bpm)")
```

Comment:
```suggestion
    local bpm_ptr = buf(0x92,2)
    local bpm = bpm_ptr:uint() 
    local bpm_display = "Unknown"
    if bpm ~= 0xFFFF then
      bpm_display = bpm / 10
    end
    subtree:add(pdj_status_f.t0a_bpm, bpm_ptr, bpm, nil, "(" .. bpm_display .. " bpm)")
```

Handle unknown BPM, otherwise will show `655.35 bpm`